share/spack/bash/spack-completion.in

```diff
@@ -0,0 +1,194 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```

Comment:
How do we get GitHub/vim to recognize this filetype, add a shebang or use `.gitattributes`?

Reply:
`.gitattributes` works for GitHub but not for vim. `# vim: set filetype=sh :` didn't work for me locally, but maybe I don't have modelines enabled. A shebang worked, but since this script is sourced, I'm not sure if it should have a shebang.

Reply:
We could also replace the extension with `.bash` or `.sh`, but then it isn't clear what the difference is between this file and the actual file you source.